server/app/services/patient_service.py

```diff
@@ -68,7 +68,7 @@ async def scheduleAppointment(self, email: str, appointmentDetails: Appointment)
         response = {"message": "Appointment updated successfully", "currentAppointment": updated_appointments}
         return 201, response
     
-    async def set_address_for_patient(self, patient_email: str, address: Location):
+    async def set_location_for_patient(self, patient_email: str, address: Location):
```

Comment:
can we keep this consistent, during function call we use location, and here we use address as args variable name